lib/net/ldap/error.rb

```diff
@@ -0,0 +1,31 @@
+class Net::LDAP
+  class Error < StandardError; end
```

Comment:
@mtodd good eye. What about having `Error` inherit from `LdapError` for a few point releases so that we can add a deprecation warning?

```ruby
class Net::LDAP
  class LdapError < StandardError
    def message
      "Deprecation warning... " + super
    end
  end
  
  class Error < LdapError; end
end
```

This allows us to eventually remove `LdapError`.